Library/Formula/kapacitor.rb

```diff
@@ -0,0 +1,209 @@
+      system "go", "install",
+             "-ldflags", "-X main.version=#{version} -X main.commit=#{revision}",
+             "./..."
+    end
+
+    inreplace kapacitor_path/"etc/kapacitor/kapacitor.conf" do |s|
+      s.gsub! "/var/lib/kapacitor", "#{var}/kapacitor"
+      s.gsub! "/var/log/kapacitor", "#{var}/log"
+    end
+
+    bin.install "bin/kapacitord"
+    bin.install "bin/kapacitor"
+    etc.install kapacitor_path/"etc/kapacitor/kapacitor.conf" => "kapacitor.conf"
+
+    (var/"kapacitor/replay").mkpath
+    (var/"kapacitor/tasks").mkpath
+  end
+
+  plist_options :manual => "kapacitord -config #{HOMEBREW_PREFIX}/etc/kapacitor.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <dict>
+          <key>SuccessfulExit</key>
+          <false/>
+        </dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/kapacitord</string>
+          <string>-config</string>
+          <string>#{HOMEBREW_PREFIX}/etc/kapacitor.conf</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{var}</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/kapacitor.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/kapacitor.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    logo = <<-EOS.undent
+      '##:::'##::::'###::::'########:::::'###:::::'######::'####:'########::'#######::'########::
+       ##::'##::::'## ##::: ##.... ##:::'## ##:::'##... ##:. ##::... ##..::'##.... ##: ##.... ##:
+       ##:'##::::'##:. ##:: ##:::: ##::'##:. ##:: ##:::..::: ##::::: ##:::: ##:::: ##: ##:::: ##:
+       #####::::'##:::. ##: ########::'##:::. ##: ##:::::::: ##::::: ##:::: ##:::: ##: ########::
+       ##. ##::: #########: ##.....::: #########: ##:::::::: ##::::: ##:::: ##:::: ##: ##.. ##:::
+       ##:. ##:: ##.... ##: ##:::::::: ##.... ##: ##::: ##:: ##::::: ##:::: ##:::: ##: ##::. ##::
+       ##::. ##: ##:::: ##: ##:::::::: ##:::: ##:. ######::'####:::: ##::::. #######:: ##:::. ##:
+      ..::::..::..:::::..::..:::::::::..:::::..:::......:::....:::::..::::::.......:::..:::::..::
+    EOS
+
+    (testpath/"config.toml").write shell_output("kapacitord config")
+    # inreplace testpath/"config.toml" do |s|
+    #   s.gsub! "enabled = true", "enabled = false"
+    #   s.gsub! "kapacitor/replay", testpath/"kapacitor/replay"
+    #   s.gsub! "kapacitor/tasks", testpath/"kapacitor/tasks"
+    # end
+    require "open3"
+    Open3.popen3("kapacitord", "-config", testpath/"config.toml") do |stdin, stdout, _|
+      stdin.close
+      assert_match logo, stdout.read
+    end
+    assert_equal 0, $?.exitstatus
```

Comment:
I'm maybe missing something but can't you just use `shell_output` for this too rather than `open3`?